__tests__/pages/eligibility.test.tsx

```diff
@@ -16,16 +18,20 @@ describe('index page', () => {
     useRouter.mockImplementation(() => ({
       route: '/eligibility',
       pathname: '/eligibility?income=20000',
-      query: '',
+      query: { income: '20000' },
       asPath: '',
     }))
   })
 
   it('should render the eligibility page', async () => {
+    const res = await mockPartialGetRequest({
+      income: '20000' as unknown as number,
```

Comment:
could just change this to 20000 and take out the type stuff? Seems to work on my end when I try it.